setup.cfg

```diff
@@ -220,6 +218,8 @@ enable =
 	bad-inline-option,
 	deprecated-pragma,
 disable = 
+	unused-import,
```

Comment:
I think is better if you disable this directly on the file and not globally

Reply:
when i tried, it didn't work

Reply:
how did you do it?

Reply:
with this comment # pylint: disable=unused-import